libs/wingsdk/src/shared-aws/table.inflight.ts

```diff
@@ -67,7 +67,19 @@ export class TableClient implements ITableClient {
     if (result.Item) {
       return unmarshall(result.Item) as Json;
     }
-    return {} as Json;
+    throw new Error(`Row does not exist (key=${key}).`);
+  }
+
+  public async tryGet(key: string): Promise<Json | undefined> {
+    const command = new GetItemCommand({
```

Comment:
No chance that an exception is thrown if key does not exist ? In other words, this.client.send will not throw an exception? I think that the answer is no based on the previous implementation of get method, but asking just in case.